packages/zoe/test/unitTests/contracts/test-priceAggregator.js

```diff
@@ -593,25 +610,19 @@ test('quoteAtTime', async t => {
   const pa = E(aggregator.publicFacet).getPriceAuthority();
 
   const quoteAtTime = E(pa).quoteAtTime(
-    7n,
+    toTS(7n),
     AmountMath.make(brandIn, 41n),
     usdBrand,
   );
 
   /** @type {PriceQuote | undefined} */
   let priceQuote;
-  quoteAtTime.then(
-    result => (priceQuote = result),
-    reason =>
-      t.notThrows(() => {
-        throw reason;
-      }),
-  );
+  t.notThrowsAsync(quoteAtTime.then(result => (priceQuote = result)));
```

Comment:
I believe that `t.notThrowsAsync` returns a Promise.. does this need to have an `await`? I know it was missing on the original, and maybe it needed to set up an assertion that would run at some point in the future, and couldn't afford to wait because it hadn't yet started the action that would eventually allow the `quoteAtTime` to resolve. But it feels like there's going to be an ungrounded promise here, or maybe if `quoteAtTime` throws then we'll get an unhandled rejection error, instead of a proper test failure.

This is probably complicated by the "manual timer is doing `waitUntilQuiescent` to allow all promises to settle" thing. But it would be nice if it could use some simpler `const priceQuote = await quoteAtTime`.

Maybe

Reply:
I wondered the same thing, and couldn't definitely establish whether or not the omission is intentional so I left it as-is. At any rate, there are enough following `await`s for this not to break.